GlideRecord/List of Child Incidents/listofchildIncident.js

```diff
@@ -0,0 +1,14 @@
+var parentID = '9c573169c611228700193229fff72400'; // This should be stored in sys_property instead of hardcoding the sys_id. 
+var inc = new GlideRecord('incident');
+inc.addEncodedQuery('parent_incident='+parentID); // sys_id of parent Incident
+inc.query();
+
+if(inc.next()){
+gs.info('List of Child Incidents of:'+inc.getDisplayValue('parent_incident'));
+gs.info(inc.getValue('number'));
```

Comment:
What is this line here?

Reply:
Should be removed right?

Reply:
As we are using next(), it is already iterating a record so have to use it to display. Removing it will display 1 record less. I have tried and after removing instead of 4 child incident, it will show 3 only. So I believe we have to keep it.

Reply:
Oh yes you're right. Might be more elegant adding all the numbers to an array and printing it then, but nevermind.